curriculum/challenges/english/02-javascript-algorithms-and-data-structures/es6/write-concise-declarative-functions-with-es6.english.md

```diff
@@ -53,6 +53,15 @@ console.log(bicycle.gear);
 
 </div>
 
+### After Test
```

Comment:
```suggestion
### Before Test
```
Similarly to #35899 I think this should reflect the fact it's setup, but it doesn't currently matter.

Reply:
It does not matter which section it goes in for this, so After Test and `tear-down` are fine.